pkg/store/driver/txn/txn_driver.go

```diff
@@ -200,6 +214,9 @@ func (txn *tikvTxn) GetMemBuffer() kv.MemBuffer {
 }
 
 func (txn *tikvTxn) SetOption(opt int, val any) {
+	if intest.InTest {
```

Comment:
Why just check in tests?

Reply:
I suppose the current implementation is ok, if this check only works in tests, we can just panic rather than returning and checking error. Another reason, compile with `intest.InTest` won't impact the performance of release builds.